app/assets/javascripts/govuk_publishing_components/analytics.js

```diff
@@ -18,3 +18,4 @@
 //= require ./analytics/explicit-cross-domain-links
 //= require ./analytics/track-click
 //= require ./analytics/track-select-change
+//= require ./analytics/cross-domain-tracking
```

Comment:
If the scripts are not sensitive to order would be good to order them alphabetically; if they're sensitive we may need a comment to state this

Reply:
I think they are sensitive to order (I remember trying to do `require_tree` at some point and it failing) although I don't know what the order is - I think some of these scripts might need to be executed before the others, but some might not.

I put this one at the bottom because it was being executed in static after this analytics file was called. Have added a comment.